embassy-stm32/src/can/fd/filter.rs

```diff
@@ -0,0 +1,379 @@
+//! Definition of Filter structs for FDCAN Module
+//! Note: This file is copied and modified from fdcan crate by Richard Meadows
```

Comment:
make this a regular comment instead of a doc comment. Keeping credits in the code is nice, but the user who's browsing the docs doesn't care about it, they just want to get their FDCAN widget working.